resources/repos.json

```diff
@@ -841,9 +841,11 @@
                 "1": "1.15.x, 1.16.x"
             },
             "CS-CoreLib Version": {
+                "10": "<em>This was never really required!</em>",
                 "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/CS-CoreLib/master/#90\">dev #90</a>"
             },
```

Comment:
If it wasn't ever required anyway, you can just remove this whole dependency in my opinion.
```suggestion
```